src/scss/main.scss

```diff
@@ -16,6 +16,9 @@
 
 // Home page sections styles.
 @import 'sections/hero/hero';
+
+//sections
+@import 'sections/about/about';
 @import 'sections/brands/brands';
 @import 'sections/plan/plan';
```

Comment:
Why last section is included first? Please make the right order.
About goes after Plan, right?

Reply:
### **Okey**